server/fleet/app.go

```diff
@@ -52,6 +53,9 @@ type AppConfigService interface {
 
 	// License returns the licensing information.
 	License(ctx context.Context) (*LicenseInfo, error)
+
+    // FleetConfig return the config.FleetConfig instance.
```

Comment:
This comment looks slightly misaligned, are you running gofmt on save?

Reply:
Hmm, we need to get `golangci-lint` running on PRs to catch things like this.